packages/openapi-react-query/.npmignore

```diff
@@ -0,0 +1,3 @@
+examples
```

Comment:
nit: this folder doesn’t exist: 

```suggestion
```

But it’s not causing any problems. Running `npm publish --dry-run` I see:

```
npm notice 4.2kB CONTRIBUTING.md
npm notice 2.2kB README.md
npm notice 322B biome.json
npm notice 172.3kB dist/index.cjs
npm notice 2.7kB dist/index.d.ts
npm notice 1.8kB dist/index.js
npm notice 1.7kB dist/index.js.map
npm notice 2.4kB package.json
npm notice 4.9kB src/index.ts
npm notice 57B tsconfig.build.json
npm notice 356B tsconfig.json
```

We may or may not need to include `biome.json`, and it’s debatable whether the TSConfigs need to stay. But the other packages do that too. If you have any opinions, open to suggestions (but by default, a sensible default is “don’t ship it to npm unless it has value;” some of the other packages just have older `.npmignores` that accidentally let biome.json and others slip through and probably need to be updated)!

Reply:
I am more into the "Save bandwidth until we can". I would even remove the `src` folder. If anyone need to run tests, read source code, the repository is open-source.

If you are also fine with that, I'll update the other packages in this PR